index.html

```diff
@@ -0,0 +1,369 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <link rel="stylesheet" href="index.css">
+  <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+  <meta charset="utf-8">
+  <title>DESIGN TEARDOWN</title>
+</head>
+
+<body>
+
+  <header>
+    <nav>
+      <div class="headerBox">
+        <h3>Smashing Magazine</h3>
+      </div>
+
+      <div class="headerBox">
+        <h3>Articles</h3>
+      </div>
+
+      <div class="headerBox">
+        <h3>Books</h3>
+      </div>
+
+      <div class="headerBox">
+        <h3>Events</h3>
+      </div>
+
+      <div class="headerBox">
+        <h3>Jobs</h3>
+      </div>
+
+      <div class="headerBox">
+        <h3>Membership</h3>
+      </div>
+
+      <div class="headerBox">
+        <h3>TOPICS</h3>
+      </div>
+    </nav>
+
+  </header>
+
+
+  <main>
+    <section id="firstSection">
+
+      <div class="firstSectionBox fs1">
+        <h1>Created To Make You Think: Meet Our New Printed Magazine</h1>
+      </div>
+
+      <div class="firstSectionBox fs2">
+        <h1>Designing And Building A Progressive Web Application Without A Framework (Part 1)</h1>
+      </div>
+      <div class="firstSectionBox fs3">
+        <h1>The Essential Guide To JavaScript’s Newest Data Type: BigInt</h1>
+      </div>
+      <div class="firstSectionBox fs4">
+        <h1>The Essential Guide To JavaScript’s Newest Data Type: BigInt</h1>
+      </div>
+      <div class="firstSectionFloat">
+        <p id="titleFloat">Don’t Miss These Latest<br>Articles</p>
+      </div>
+    </section>
+
+    <section id="secondSection">
+      <div class="secondSectionBox1">
```

Comment:
`secondSectionBox1` is one-third-width container which should have
- [ ] Three `<article>` with a width of `33%` (as seen on the smashing magazine webpage) -- There is extra space appearing at the end of this container.
- [ ] Each `<article>` should have a title and paragraph with placeholder text